docs/en/build_and_run.md

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
The English docs are name as xxx_en.md now, so build_and_run_en.md is better

Reply:
fix